cips/versioned_gas_scheduler_variables.md

```diff
@@ -0,0 +1,60 @@
+| cip | 24 |
+| - | - |
+| title | Versioned Gas Scheduler Variables |
+| description | Transition to hard fork-only updates for gas scheduler variables |
+| author | Nina Barbakadze (@ninabarbakadze) |
+| discussions-to |  |
+| status | Draft |
+| type | Standards Track |
+| category | Core |
+| created | 2024-07-24 |
+
+## Abstract
+
+Introduce versioning for on-chain governance modifiable gas scheduler parameters, such as `blob.GasPerBlobByte` and `auth.TxSizeCostPerByte`, to only allow changes during hard fork upgrades.
+
+## Motivation
+
+Versioning on-chain governance modifiable parameters, such as `blob.GasPerBlobByte` and `auth.TxSizeCostPerByte`, aims to stabilize gas estimation by removing block-to-block variability.
+
+## Specification
+
+Currently, `GasPerBlobByte` and `TxSizeCostPerByte` are module parameters within the `blob` and `auth` modules, allowing for their modification via `ParameterChangeProposal`. The proposed modification changes these parameters to hardcoded constants within the application, accessible via version-specific getters.
+
+### Parameters
+
+The proposal makes these two variables modifiable through hard fork upgrades:
+
+Previously:
+
+1. `blob.GasPerBlobByte`
+2. `auth.TxSizeCostPerByte`
+
+Now:
+
+1. `appconsts.GasPerBlobByte`
+1. `appconsts.TxSizeCostPerByte`
```

Comment:
Can we include what the values of these will be when implemented (in v3 potentially)